docs/contents/dev-docs/release.md

```diff
@@ -6,10 +6,17 @@ Remember that there is no single ACK binary. Rather, when we build a release
 for ACK, that release is for one or more individual ACK service controllers
 binaries, each of which are installed separately.
 
-This documentation covers the steps involved in building a service controller's
-release artifacts, including the Helm charts and binary Docker images for an
-ACK service controller, publishing those artifacts and tagging the Git source
-repository appropriately to create an official "release".
+This documentation covers the steps involved for officially publishing
+a ACK service controller's release artifacts.
+
+Once ACK service controller changes are tested by service team and they wish to
+release latest artifacts, service team only needs to tag the service-controller
+github repository with a semver tag (Ex: v0.0.1). 
```

Comment:
From Slack: "all you would need to do is create a new release on the Github repo" - Create a release (with what as the message?) and that should subsequently create a tag.

Reply:
I would also add here some documentation linking to SemVer, so teams know when to push a patch vs. a minor release.

Reply:
For 1) I pointed that steps for cutting a release are mentioned in the document below.
For 2) That information is already present in another doc. I have linked it.

Good suggestions. Thanks.